content/_code-samples/airgapped-wallet/py/airgapped-wallet.py

```diff
@@ -0,0 +1,203 @@
+import os
+import base64
+import qrcode
+import platform
+from PIL import Image
+from pathlib import Path, PureWindowsPath, PurePath
+from cryptography.fernet import Fernet
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+from xrpl import wallet
+from xrpl.core import keypairs
+from xrpl.utils import xrp_to_drops
+from xrpl.models.transactions import Payment
+from xrpl.transaction import safe_sign_transaction
+
+
+def create_wallet():
+    """
+    Generates a keypair
+    """
+
+    seed = keypairs.generate_seed()
+    pub, priv = keypairs.derive_keypair(seed)
+
+    address = keypairs.derive_classic_address(pub)
+    private = keypairs.derive_classic_address(priv)
```

Comment:
Is `derive_classic_address(priv)` the right function here? (I thought `priv` was already the private key?)

Reply:
You're right, my mistake